libs/standard-tests/langchain_standard_tests/integration_tests/chat_models.py

```diff
@@ -1,74 +1,31 @@
 import json
-from abc import ABC, abstractmethod
-from typing import Type
 
 import pytest
 from langchain_core.language_models import BaseChatModel
 from langchain_core.messages import AIMessage, AIMessageChunk, HumanMessage, ToolMessage
-from langchain_core.pydantic_v1 import BaseModel, Field
-from langchain_core.tools import tool
 
+from langchain_standard_tests.unit_tests.chat_models import (
+    ChatModelTests,
```

Comment:
why subclass? Can't we treat them separately?

If anything should be shared, maybe do a base class?

Reply:
that is what this is

Reply:
see ChatModelUnitTests for unit tests

Reply:
lol facepalm